Incorporate no_docs_url option and update CI tests (#4)
xlab-steampunk/spotter-action: README.md

| `upload_values`   | no       | false   | Parses and uploads values from Ansible task parameters to the backend.                                                                                                             |                                
| `upload_metadata` | no       | false   | Uploads metadata (i.e., file names, line and column numbers) to the backend.                                                                                                       |                                
| `display_level`   | no       | hint    | Displays check results with specified level or greater (e.g., warning will show all warnings and errors, but suppress hints). Available options: hint, warning, error.             |

> | `no_docs_url`     | no       | false   | Disables outputting URL to documentation.                                                                                                                                          |  

Reviewer: ```suggestion
| `no_docs_url`     | no       | false   | Omits documentation URLs from the output.                                                                                                                                          |  
```

Author: Done.
